### DISCUSSION TITLE
Publishing papers in other formats with Typst

### DISCUSSION THREAD

**Discussion by Jegp** [typst/typst#4099] [General]
I'm absolutely awed by the Typst project. From the interface to the implementation, it just seems like rock-solid work. Well done!

But. Most of the journals/venues I'm targeting are so heavily invested into docx or LaTex that they don't support other formats. And from what I can gather, Typst doesn't export to either of them at the moment. How do you get around that? Is Typst a reasonable bet to use for academic publications?

**Comment by Enivex** [comment]
> I'm absolutely awed by the Typst project. From the interface to the implementation, it just seems like rock-solid work. Well done!
> 
> But. Most of the journals/venues I'm targeting are so heavily invested into docx or LaTex that they don't support other formats. And from what I can gather, Typst doesn't export to either of them at the moment. How do you get around that?

Pandoc has typst support, though it's very lossy. Not something that can be relied upon without heavy postprocessing.

> Is Typst a reasonable bet to use for academic publications?

Realistically not for anything that requires the document source and not just the pdf.
